tools/peptideshaker/peptide_shaker.xml

```diff
@@ -154,6 +154,10 @@
 
         ## # ls -l \$cwd/output_reports/* ;
 
+        #if '11' in str($outputs).split(','):
+            find \$cwd/output_reports -name '*Extended_PSM_Report*' -exec bash -c 'mv "$0" "psmx.txt"' {} \;
```

Comment:
general Q, do you think we can get command line arguments into the CLI, so that this hack is not needed anymore and we know the path to the output files?